pkg/ast/spl/spl.peg

```diff
@@ -2047,39 +2048,28 @@ TransactionBlock <- PIPE CMD_TRANSACTION txnOptions:TransactionOptions {
 
 // The List of fields should Always be given at the last.
 // The TransactionDefinitionOptionsList will consist of all the Transaction Options
-TransactionOptions <- txnOptions:TransactionDefinitionOptionsList? SPACE? fields:FieldList?  {
+TransactionOptions <- txnOptions:TransactionDefinitionOptionsList?  {
 
     transactionRequest := &structs.TransactionArguments{
         Fields:     nil,
         StartsWith: "",
         EndsWith:   "",
     }
 
-    if fields != nil {
-        transactionRequest.Fields = fields.([]string)
-    }
+    //if fields != nil {
+      //  transactionRequest.Fields = fields.([]string)
+    //}
```

Comment:
nit: for the final version of the PR, delete these lines if they're not needed

Reply:
Yes, I will delete them!